std/typecons.d

```diff
@@ -4742,12 +4855,14 @@ if (is(T == class) || is(T == interface))
 }
 
 /**
- * Supports structural based typesafe conversion.
- *
- * If $(D Source) has structural conformance with the $(D interface) $(D Targets),
- * wrap creates internal wrapper class which inherits $(D Targets) and
- * wrap $(D src) object, then return it.
- */
+Supports structural based typesafe conversion.
+
+If `Source` has structural conformance with the `interface` `Targets`,
+wrap creates internal wrapper class which inherits `Targets` and
```

Comment:
"creates _an_"
"and _wraps_ _the_ `src` object, then _returns_ it."

Would fix them myself, but on my phone.